pkg/networkpath/traceroute/common/common_windows.go

```diff
@@ -0,0 +1,184 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package common contains common functionality for both TCP and UDP
+// traceroute implementations
+package common
```

Comment:
Can we move this to a file that is more descriptive of its contents? e.g. `raw_sock_windows.go`.

Reply:
Or even its own package?